imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -171,14 +204,46 @@ def calculate_calibration_factor(time: int) -> None:
 
     Parameters
     ----------
-    time : int
-        Input time.
-    """
-    # NOTE: waiting on fake calibration data to write this.
-    pass
-
+    acquisition_time : float
+        Acquisition time in S/C MET.
 
-def apply_in_flight_calibration(data: np.ndarray) -> None:
+    Returns
+    -------
+    cal_factor : numpy.ndarray
+        Calibration factor for each 7 CEM detectors. Array shape is (7,).
+    """
+    # Read in in-flight calibration data
+    in_flight_cal_df = read_in_flight_cal_data()
+    # For the given acquisition time, find it's nearest pre and post time
+    pre_cal_time = (
+        in_flight_cal_df[in_flight_cal_df["met_time"] < acquisition_time]
```

Comment:
Should one of the checks include equivalence such as:
```suggestion
        in_flight_cal_df[in_flight_cal_df["met_time"] <= acquisition_time]
```

Reply:
Based on my conversation with Ruth, it sounds like they won't have time that would equal pre or post time. They won't be collecting science data during calibration data collection. But I will double check with her. Capturing that question on this ticket https://github.com/IMAP-Science-Operations-Center/imap_processing/issues/1232

Reply:
Ruth confirmed that we don't need to use `=` comparison given above reason.